src/config/index.ts

```diff
@@ -120,4 +121,5 @@ export const APP_CONFIG: AppConfig = {
     component: Analytics(),
     eventCb,
   },
+  knownAddresses: new Map<Address, { name: string; iconUrl?: string }>([]),
```

Comment:
Would be great to see an example in here for the demo, for example set:

```
 "0xCC3E7c85Bb0EE4f09380e041fee95a0caeDD4a02": {
    name: "Compound DAO",
    iconUrl: "/compound-dao.png"
}
```

Can add this icon to public folder: https://github.com/papercliplabs/compound-blue/blob/main/public/identity/compound.png